contrib/chart/backstage/templates/backstage-app-config.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "backstage.fullname" . }}-app
+data:
+  APP_CONFIG_app_baseUrl: {{ .Values.appConfig.app.baseUrl | quote | quote }}
```

Comment:
Could we not template this to be an `app-config.<smth>.yaml` instead? seems simpler and more readable that going through env vars.

Reply:
As per discussion in discord I moved most of the config to app-config.local.yaml but I kept a minimal configuration as environment variables as this was needed for the frontend sample app.